src/native/minipal/cpufeatures.c

```diff
@@ -385,6 +385,14 @@ int minipal_getcpufeatures(void)
                 result |= XArchIntrinsicConstants_AvxVnni;
             }
 
+            if ((cpuidInfo[CPUID_EDX] & (1 << 4)) != 0)                                                        // AVX-VNNI-INT8
+            {
+                if ((cpuidInfo[CPUID_EDX] & (1 << 10)) != 0)
+                {
+                    result |= XArchIntrinsicConstants_AvxVnniInt;                                              // AVX-VNNI-INT16
+                }
+            }
```

Comment:
nit: The existing pattern here is:
```suggestion
            if (((cpuidInfo[CPUID_EDX] & (1 << 4)) != 0) &&                                                    // AVX-VNNI-INT8
                ((cpuidInfo[CPUID_EDX] & (1 << 10)) != 0))                                                     // AVX-VNNI-INT16
            {
                result |= XArchIntrinsicConstants_AvxVnniInt;
            }
```